src/modules/site-v2/base/views/data/data.py

```diff
@@ -8,22 +8,28 @@
 from config     import config
 
 from caendr.models.error           import EnvVarError, FileUploadError
-from caendr.models.datastore       import Species, TraitFile
+from caendr.models.datastore       import Species, TraitFile, User
+from caendr.models.status          import PublishStatus
+from caendr.models.trait           import Trait
 from caendr.api.phenotype          import get_phenotype_values_for_trait
+from caendr.services.cloud.secret  import get_secret
 from caendr.services.cloud.storage import get_blob, download_blob_to_file
 from caendr.services.logger        import logger
 from caendr.services.validate      import validate_file, StrainValidator, NumberValidator
 from caendr.utils.local_files      import LocalUploadFile
 from caendr.utils.env              import get_env_var
 from caendr.utils.data             import get_file_format
-from base.utils.auth               import jwt_required, get_current_user, user_is_admin
-from base.utils.trait              import add_trait, update_trait_metadata, user_is_trait_owner
-from base.forms                    import TraitSubmissionForm, EmptyForm
+from base.utils.auth               import jwt_required, get_current_user, user_is_admin, check_feature_flag
+from base.utils.trait              import add_trait, update_trait_metadata
+from base.utils.view_decorators    import parse_trait
+from base.forms                    import TraitSubmissionForm, EmptyForm, format_form_errors
 from constants                     import TOOL_INPUT_DATA_VALID_FILE_EXTENSIONS
 
 MODULE_DB_OPERATIONS_BUCKET_NAME = get_env_var('MODULE_DB_OPERATIONS_BUCKET_NAME')
 UPLOADS_DIR = os.path.join('.', 'uploads')
 
+SITE_OWNER = User.get_ds(get_secret('SITE_OWNER_USER_ID'))
```

Comment:
consider adding a graceful fallback if the SITE_OWNER returns `None` (in case the data store is not populated, etc).